common/app/model/EmailNewsletters.scala

```diff
@@ -296,9 +296,9 @@ object EmailNewsletters {
     teaser = "Uncover unconventional destinations and rediscover old favourites - let our travel editors guide you to trips worth taking",
     description = "Weekly travel inspiration. Off-piste attractions, budget breaks, top 10s and reader reviews. Uncover unconventional destinations and rediscover old favourites - let our travel editors guide you to trips worth taking",
     frequency = "Every Wednesday",
-    listId = 2211,
+    listId = 3806,
     tone = Some("feature"),
-    aliases = List(3806, 3807),
+    aliases = List(2211, 3807),
     signupPage = Some("/travel/2016/aug/18/sign-up-to-the-flyer"),
     exampleUrl = Some("/email/the-flyer?format=email-connected")
```

Comment:
So in `newsletters.scala`, the exampleUrl was set to a default, and then [there was a function](https://github.com/guardian/frontend/pull/16562/files#diff-9d741a60bc0d1397c123deea81ec79bcL37) in the ab test that set the exampleUrl to the correct preview according to the user test bucket.

The ab test is being deleted, so there is nothing to change this now... so everyone will see the  `?format=email-connected` preview. Not sure how much of an issue it is? (minor?) 😁